notebooks/demo-train-with-faq.ipynb

```diff
@@ -0,0 +1,1001 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    eval_dataset['generated_answers'] = generated_answers
</pre><p>@Surya, I realized the code to split _source_ from the generated answer was removed here. In metrics generation that will play a role. Is there something I'm missing here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>